activities.el

```diff
@@ -495,6 +530,59 @@ accordingly."
       (setf activities-mode-timer nil))
     (remove-hook 'kill-emacs-hook #'activities-mode--killing-emacs)))
 
+(require 'easymenu)
+
+(setq activities-menu-item-resume
+      '("Resume..."
+        :help "Resume an existing activity"
+        :filter (lambda (&optional _)
+                  (let ((current-activity-name
+                         (when-let ((current-activity (activities-current)))
+                           (activities-activity-name current-activity))))
+                    (mapcar (lambda (act)
+                              (vector act `(activities-resume (activities-named ,act))
+                                      :style 'radio
+                                      :selected (equal current-activity-name act)
+                                      ))
+                            (activities-names))))
+        ))
```

Comment:
I'm not sure what this is about, but a bare `setq` on an undefined variable is bogus, and there are hanging parens.  :)

Reply:
Done. Now a defvar.